package.json

```
@@ -1118,56 +1118,94 @@
-        "title": "Connect to Kafka Clusters",
+        "title": "Get Started with Confluent for VS Code",
         "description": "Interact with Apache Kafka® clusters running locally or on Confluent Cloud.",
         "steps": [
           {
-            "id": "connect-local-cluster",
-            "title": "Connect to Kafka",
-            "description": "**Confluent Cloud Clusters** \nLog in to your Confluent account to see your Confluent Cloud resources. \n[Log in](command:confluent.connections.ccloud.signIn)\n[Sign up for free](https://confluent.cloud/signup?utm_source=vscode-ext)\n**Local Clusters** \nIf you have Kafka running locally, it appears automatically in the Resources view. \n[Start Local Resources](command:confluent.docker.startLocalResources)",
+            "id": "connect-conflulent-cloud",
+            "title": "Connect to Confluent Cloud",
+            "description": "Sign in to see your Confluent Cloud resources.\n[Sign in](command:confluent.connections.ccloud.signIn)\n[Sign up for free](https://confluent.cloud/signup?utm_source=vscode-ext)",
             "media": {
               "image": "resources/walkthrough/connect.png",
               "altText": "Connect to Kafka clusters"
             },
             "completionEvents": [
-              "onCommand:command:confluent.connections.ccloud.signIn",
-              "onCommand:confluent.docker.startLocalResources"
+              "onCommand:command:confluent.connections.ccloud.signIn"
             ]
           },
           {
             "id": "create-connections",
-            "title": "Connect directly to Kafka / Schema Registry",
-            "description": "To connect to an Apache Kafka or Schema Registry cluster, click $(plus) next to Resource > Other. The cluster can be in Confluent Cloud, Confluent Platform, or hosted elsewhere.\n[Open Connection Form](command:confluent.connections.direct)",
+            "title": "Connect Directly",
+            "description": "To connect to Confluent Cloud (via API Credential), Confluent Platform, WarpStream, Apache Kafka, and/or Schema Registry clusters.\n Select the $(plus) icon at the top of the Resources view to create or import a connection.\n[Create Connection](command:confluent.connections.direct)",
             "media": {
               "image": "resources/walkthrough/direct-connection-form.png",
-              "altText": "Create a new connection to a Kafka cluster or Schema Registry"
+              "altText": "Create a new connection to a Kafka cluster and/or Schema Registry"
             },
             "completionEvents": [
               "onCommand:confluent.connections.direct"
             ]
           },
+          {
+            "id": "connect-local-cluster",
+            "title": "Start Local Environment",
+            "description": "Start Kafka and Stream Registry clusters from within VS Code using the extension in the Resources view.\n[Start Local Resources](command:confluent.docker.startLocalResources)",
```

Probably don't need to mention VS Code and the extension since this is a walkthrough for the extension itself, right?
```suggestion
            "description": "Start Kafka and Stream Registry clusters by clicking the $(debug-start) icon next to the 'Local' item in the Resources view.\n[Start Local Resources](command:confluent.docker.startLocalResources)",
```